### ISSUE TITLE
Failing test on alpha branch

### ISSUE THREAD

**Issue by edjames** [vzaar/vzaar-api-ruby#17]
Hi vzaar

On the latest version of the `version-2.0.0-alpha` branch I'm getting a failing spec:

```
1) VzaarApi::Playlist.find when the playlist can be found finds the playlist
     Failure/Error:
       expect(playlist.embed_code).to match(
         "<iframe id=\"vzpl-12692\" name=\"vzpl-12692\" " \
         "title=\"vzaar video player\" class=\"vzaar video player\" " \
         "type=\"text/html\" width=\"927\" height=\"340\" frameborder=\"0\" " \
         "allowFullScreen allowTransparency=\"true\" mozallowfullscreen " \
         "webkitAllowFullScreen src=\"//#{api_envs["vd_hostname"]}/playlists/12692\">" \
         "</iframe>"
       )

       expected "<iframe id=\"vzpl-12692\" name=\"vzpl-12692\" title=\"vzaar video player\" class=\"vzaar video playe..." mozallowfullscreen webkitAllowFullScreen src=\"//view.vzaar.localhost/playlists/12692\"></iframe>" to match "<iframe id=\"vzpl-12692\" name=\"vzpl-12692\" title=\"vzaar video player\" class=\"vzaar video player\" type=\"text/html\" width=\"927\" height=\"340\" frameborder=\"0\" allowFullScreen allowTransparency=\"true\" mozallowfullscreen webkitAllowFullScreen src=\"///playlists/12692\"></iframe>"
     # ./spec/vzaar_api/playlist_spec.rb:22:in `block (5 levels) in <module:VzaarApi>'
     # ./spec/vzaar_api/playlist_spec.rb:9:in `block (4 levels) in <module:VzaarApi>'
```

I'm wondering if this is a known issue and something I'm missing in my own environment, of if this is a bug.

Ed.

**Comment by alol** [comment]
Hi Ed, sorry for the delayed response. This has now been fixed as of 0e00d12, and you'll now note the alpha branch is now the master branch.

**Comment by alol** [comment]
Also: many thanks for the bug report.
